pb-crm-ui/src/app/contact-book/components/contact-book-home/contact-book-home.component.ts

```diff
@@ -80,24 +104,24 @@ export class ContactBookHomeComponent implements OnInit, AfterViewInit {
 
   createContract(contact: ContactModel) {
     this.contactCache.addContact(contact).subscribe(contact => {
-      this.snackbar.open('Contact Created', 'Close', {duration: 1000});
+      this.snackbar.open('Contact Created', 'Close', { duration: 1000 });
       this.setViewContact(contact);
     });
   }
 
   editContact(contact: ContactModel) {
     this.contactCache.updateContact(contact).subscribe(updatedContact =>
-      this.setViewContact(updatedContact)
+      this.setViewContact(updatedContact),
```

Comment:
This comma is not supposed to be there and is invalid syntax. Interesting it even compiles.